spec/models/embed/purl_spec.rb

```diff
@@ -99,6 +99,13 @@
     end
   end
 
+  describe 'size' do
```

Comment:
Can you extract:
```ruby
let(:purl) { described_class.new('12345') }
subject { purl.size } 
before do
  stub_purl_response_with_fixture(multi_file_purl)
end
it { is_expected.to eq 12_345 * 2 }
```

On these? I think that helps the reader because it highlights the subject of the test and separates out the setup for the test.

Reply:
@jcoyne I think I understand but it would help me to walk through this one with you if you are available Weds AM (EST)?

Reply:
@jcoyne I've updated the setup for these tests--can you take a look and see if I did what you were expecting?